src/http_request.cpp

```diff
@@ -105,20 +105,31 @@ const http::header_view_map http_request::get_cookies() const {
 }
 
 std::string_view http_request::get_arg(std::string_view key) const {
```

Comment:
Since get_arg() now returns only the first matching item (which does keep the API the same), should there also be a get_args() method that returns a `std::span<const std::string>` or whatever type makes the least memory allocations?  IIRC a `span<const T>` can be made from a `vector<T>` implicitly.

Reply:
afaik span is C++20, so this would require bumping the min version from 17.